README.adoc

```diff
@@ -13,7 +13,7 @@ If you are an administrator, then read the https://kubesaw.github.io/ksctl-cheat
 
 == Build
 
-Requires Go version 1.20.x - download for your development environment https://golang.org/dl/[here].
+Requires Go version 1.21.x - download for your development environment https://golang.org/dl/[here].
```

Comment:
Minor - if we can keep Readme, same in all repos. 
```suggestion
Requires Go version 1.21.x (1.21.13 or higher) - download for your development environment https://golang.org/dl/[here].
```

Reply:
done in [8c45a63](https://github.com/kubesaw/ksctl/pull/103/commits/8c45a63014cc61297becbf47ae0a10dbffde38cc)